sdk/python/feast/infra/offline_stores/file_source.py

```diff
@@ -180,19 +179,20 @@ class FileOptions:
     def __init__(
         self,
         file_format: Optional[FileFormat],
-        file_url: Optional[str],
         s3_endpoint_override: Optional[str],
+        uri: Optional[str],
```

Comment:
Wanted to confirm that this is not user facing?

Reply:
Yup isn't called anymore but maybe @adchia can also confirm?